src/encoded/static/components/app.js

```diff
@@ -212,6 +212,26 @@ EulaModal.propTypes = {
     signup: PropTypes.func.isRequired,
 };
 
+const AccountCreationFailedModal = ({ closeModal, date }) => (
+    <Modal>
+        <ModalHeader title="Failed to create a new account." closeModal={closeModal} />
+        <ModalBody>
+            <p>
+                Creating a new acount failed. Please, contact <a href={`mailto:encode-help@lists.stanford.edu?subject=Creating e-mail account failed&body=Creating an account failed at time: ${date}`}>support</a>.
```

Comment:
"acount" should be spelled "account". Also I think "Please" doesn't need a comma after it. It could just be "Please contact support."

Reply:
Done.